PYME/Acquire/Hardware/driftTracking.py

```diff
@@ -125,9 +125,9 @@ def __init__(self, scope, piezo=None, frame_source=None):
         if frame_source is None:
             self.frame_source = StandardFrameSource(scope.frameWrangler)
         
-        self.focusTolerance = .05 #how far focus can drift before we correct

```

Comment:
These defaults were carefully chosen through a lot of trial and error on a couple of systems. I would be reluctant to make hard-coded changes to them, although would entertain/support shifting them to defaults in `__init__()` so that setup-specific alternative defaults could be specified.

Notably:

focusTolerance - setting this too low results in instability on systems with weak illumination for the tracking camera / a noisy signal.

deltaZ - the value of 0.2 is based on Nyquist sampling of the PSF in z for a ~1.4NA objective. It is already a bit optimistic (especially if you are going to decrease the focus tolerance), as we assume that signal intensity varies linearly with z between calibration steps. I really wouldn't want to increase it much unless you were using a low NA objective - especially if also decreasing focusTolerance.

stackHalfSize - this is based on being able to do z-stepped imaging of an entire cell nucleus (around 10-12 um) and have enough leeway at each end so that you can still correct. I can see an argument for decreasing this for some use cases to make things faster, but think this is a pretty good default for having something which mostly works out of the box.